### ISSUE TITLE
Directions on how to start it the first time

### ISSUE THREAD

**Issue by base698** [ternjs/tern#710]
I see no directions how to run it in the readme.  I have the emacs plug in and cloned the server.  Did npm install, node lib/tern.js which exits with status code 0.  How do I run it?

**Comment by marijnh** [comment]
The emacs plugin will run a server automatically. Did you enable it? To start a server manually, you want `bin/tern`.

**Comment by base698** [comment]
How do you run it automatically?  I see tern-use-server in helm and a few other tern commands which all error.  Position outside of file when tern-server is set and a connection refused when it's not connected.

How do I make it aware of my js files?  I see mention of .tern-project via google file but no mention on how to configure it or a default version.

**Comment by marijnh** [comment]
Yes, that's how you configure Tern. Does that mean your issue is resolved? If not, show a project and a config file, and describe what is going wrong.
